react-toggle-switch-component/src/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import ReactDOM from 'react-dom/client';
+
+class CustomButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+
+  }
+
+  handleClick(e) {
+    const checkbox = document.querySelector('.checkbox');
+    if (checkbox.checked === true) {
+      checkbox.checked = false;
+    } else { checkbox.checked = true; }
+  }
```

Comment:
We never want to use the `document` object to select elements in React. You should be using state to set the attributes on the elements.